src/vw/src/main/scala/VowpalWabbitFeaturizer.scala

```diff
@@ -0,0 +1,140 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark
+
+import com.microsoft.ml.spark.featurizer._
+import org.apache.spark.ml.Transformer
+import org.apache.spark.ml.param.{IntParam, ParamMap, StringArrayParam}
+import org.apache.spark.sql.types.{ByteType, DoubleType, FloatType, IntegerType, LongType, ShortType, _}
+import org.apache.spark.sql.{DataFrame, Dataset, Row}
+import org.apache.spark.sql.functions.{col, struct, udf}
+import org.vowpalwabbit.spark.VowpalWabbitMurmur
+import org.apache.spark.ml.linalg.Vectors
+import org.apache.spark.ml.util.{ComplexParamsReadable, ComplexParamsWritable, Identifiable}
+import org.apache.spark.ml.linalg.SQLDataTypes.VectorType
+
+import scala.collection.mutable.ArrayBuilder
+
+object VowpalWabbitFeaturizer extends ComplexParamsReadable[VowpalWabbitFeaturizer]
+
+class VowpalWabbitFeaturizer(override val uid: String) extends Transformer
+  with HasInputCols with HasOutputCol with HasNumBits with Wrappable with ComplexParamsWritable
+{
+  def this() = this(Identifiable.randomUID("VowpalWabbitFeaturizer"))
+
+  setDefault(inputCols -> Array())
+
+  val seed = new IntParam(this, "seed", "Hash seed")
+  setDefault(seed -> 0)
+
+  def getSeed: Int = $(seed)
+  def setSeed(value: Int): this.type = set(seed, value)
+
+  val stringSplitInputCols = new StringArrayParam(this, "stringSplitInputCols",
+    "Input cols that should be split add word boundaries")
```

Comment:
what do you mean by "add word boundaries"?

Reply:
is this minor spelling mistake: add -> at?

Reply:
yeah... my spelling skill seems to deteriorate